packages/amplify-cli-core/src/help/commands-info.ts

```diff
@@ -0,0 +1,1257 @@
+        "subCommandDescription": "Scaffolds a skeleton Amplify CLI plugin",
+        "subCommandUsage": "amplify plugin init",
+        "learnMoreLink": "https://docs.amplify.aws/cli/commands/plugin",
+        "subCommandFlags": []
+      },
+      {
+        "subCommand": "configure",
+        "subCommandDescription": "Configures Amplify CLI plugin options",
+        "subCommandUsage": "amplify plugin configure",
+        "learnMoreLink": "https://docs.amplify.aws/cli/commands/plugin",
+        "subCommandFlags": []
+      },
+      {
+        "subCommand": "list",
+        "subCommandDescription": "Lists general plugin information",
+        "subCommandUsage": "amplify plugin list",
+        "learnMoreLink": "https://docs.amplify.aws/cli/commands/plugin",
+        "subCommandFlags": []
+      },
+      {
+        "subCommand": "scan",
+        "subCommandDescription": "Explicitly starts a scan/search for new and existing plugins",
+        "subCommandUsage": "amplify plugin scan",
+        "learnMoreLink": "https://docs.amplify.aws/cli/commands/plugin",
+        "subCommandFlags": []
+      },
+      {
+        "subCommand": "add",
+        "subCommandDescription": "Explicitly adds a plugin for the Amplify CLI to use",
+        "subCommandUsage": "amplify plugin add",
+        "learnMoreLink": "https://docs.amplify.aws/cli/commands/plugin",
+        "subCommandFlags": []
+      },
+      {
+        "subCommand": "remove",
+        "subCommandDescription": "Explicitly removes a plugin from the Amplify CLI",
+        "subCommandUsage": "amplify plugin remove",
+        "learnMoreLink": "https://docs.amplify.aws/cli/commands/plugin",
+        "subCommandFlags": []
+      },
+      {
+        "subCommand": "verify",
+        "subCommandDescription": "Verifies if a plugin package/directory is a valid Amplify CLI plugin",
+        "subCommandUsage": "amplify plugin verify",
+        "learnMoreLink": "https://docs.amplify.aws/cli/commands/plugin",
+        "subCommandFlags": []
+      }
+    ]
+  },
+  {
+    "command": "update",
+    "commandDescription": "Update resource for an Amplify category in your local backend",
+    "commandUsage": "amplify update <category>",
+    "learnMoreLink": "https://docs.amplify.aws/cli/commands/update",
+    "commandFlags": [],
+    "subCommands": [
+      {
+        "subCommand": "<category>",
+        "subCommandDescription": "Update resource for an Amplify category in your local backend",
+        "subCommandUsage": "amplify update <category> [flags]",
+        "learnMoreLink": "https://docs.amplify.aws/cli/commands/update",
+        "subCommandFlags": [
+          {
+            "short": "",
+            "long": "headless",
+            "flagDescription": "Headless JSON payload (see https://docs.amplify.aws/cli/usage/headless)"
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "command": "publish",
+    "commandDescription": "Executes amplify push and builds, hosts the frontend app",
```

Comment:
I think adding "builds" here is more confusing than helpful

```suggestion
    "commandDescription": "Executes amplify push and hosts the frontend app",
```